.github/workflows/ci_cron.yml

```diff
@@ -52,10 +52,10 @@ jobs:
       - name: Install Ruby ${{ matrix.ruby-version }}
         uses: ruby/setup-ruby@a6e6f86333f0a2523ece813039b8b4be04560854 # tag v1.190.0
         with:
-          ruby-version: ${{ matrix.ruby-version }}
+          ruby-version: ${{ matrix.ruby-version }}                                                                                                               
 
       - name: Set up mini-envs for ruby version
-        uses: kanga333/variable-mapper@3681b75f5c6c00162721168fb91ab74925eaebcb # tag v0.3.0
+        uses: ./.github/actions/variable-mapper # no tag - locally sourced
```

Comment:
I think you can discard these comments. They serve as a map between a sha and a tag version. The `./` local path prefix will already convey that the action is locally sourced.

Reply:
This tag was added [once upon a time](https://github.com/newrelic/newrelic-ruby-agent/blob/dev/.github/workflows/pr_closed.yml#L17) for another local action so I thought to follow that pattern! But I agree it should be implied